src/utils.ts

```diff
@@ -347,7 +347,7 @@ export const validateCrypto = () => {
   if (typeof getCryptoSubtle() === 'undefined') {
     throw new Error(`
       auth0-spa-js must run on a secure origin.
-      See https://github.com/auth0/auth0-spa-js/blob/master/FAQ.md#why-do-i-get-auth0-spa-js-must-run-on-a-secure-origin 
+      See https://github.com/auth0/auth0-spa-js/blob/master/FAQ.md#why-do-i-get-auth0-spa-js-must-run-on-a-secure-origin
```

Comment:
The test is failing because of this removal of the space at the end of the line. Can we add it back in?